src/constants/index.ts

```diff
@@ -4,16 +4,21 @@
 import BN from 'bn.js';
 import type { ApiState } from 'types';
 
+export enum LOCAL_STORAGE_KEY {
+  CUSTOM_ENDPOINT = 'contractsUiCustomEndpoint',
```

Comment:
this could just be a string const

```ts
export const LOCAL_STORAGE_KEY = 'contractsUiCustomEndpoint';
```

Reply:
Ah yes sure. I was creating this to maybe refactor this part into this enum as well? What do you think?

https://github.com/paritytech/contracts-ui/blob/9bcb6eb8f065c785adfc43b2a8e55b2f41349867/src/ui/contexts/ApiContext.tsx#L20-L23

Reply:
why an enum? we don't have a collection of storage keys, just one. even if we add more keys are better ways to represent them than an enum.

Reply:
Please check https://github.com/paritytech/contracts-ui/commit/f3fef56659765c190f9f0f19d5bbecf8c0dead09 and let me know what you think. No problem to refactor it to just hardcode the string in place. 🤷  

I don't think Enums are as bad as people make it seem. But if we don't want to use Enums, fine for me. 

As of now we have three local storage keys. These are hardcoded in the app. Which works just fine as they are not used anywhere else. 
I think a [String Enum](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) is a good way to manage the constant string values we use for the local storage.

This way we can use the enum as a type directly instead of using something like this if we use a normal object:
```typescript
const LOCAL_STORAGE_KEYS = {
  CUSTOM_ENDPOINT: 'contractsUiCustomEndpoint',
  PREFERRED_ENDPOINT: 'contractsUiPreferredEndpoint',
  THEME: 'theme',
};
type LOCAL_STORAGE_KEY =  keysof<LOCAL_STORAGE_KEYS>;
```

Or a IMHO messy union type version
```typescript
type LOCAL_STORAGE_KEY = 'contractsUiCustomEndpoint' | 'contractsUiPreferredEndpoint' | 'theme';
```

Reply:
my bad, I was only looking at this
```ts
export enum LOCAL_STORAGE_KEY {
  CUSTOM_ENDPOINT = 'contractsUiCustomEndpoint',
}
```
in this case, up to you. 

there is also the solution of exporting simple strings. 

```ts
export const CUSTOM_ENDPOINT_KEY = 'contractsUiCustomEndpoint'
export const PREFERRED_ENDPOINT_KEY = 'contractsUiPreferredEndpoint'
export const THEME_KEY = 'theme'
```
It would be nicer when accessing than writing long constructs in caps like `LOCAL_STORAGE_KEY.CUSTOM_ENDPOINT`

Reply:
Okay you are right, enums a kind of unusable with ts right now.
I've changed it to a plain old object and extracted the values from it, so now both `LOCAL_STORAGE_KEY.CUSTOM_ENDPOINT` or `contractsUiCustomEndpoint` and be used in a typesafe way.

https://github.com/paritytech/contracts-ui/pull/435/commits/1bd9de3a95db4dcc37c3e8370c9e3111e06d5fbd

<img width="618" alt="Screenshot 2023-03-22 at 10 03 33" src="https://user-images.githubusercontent.com/839848/226855378-72a48632-0a9f-4b41-b041-878109637ba0.png">